pkg/plugins/resources/k8s/store.go

```diff
@@ -231,6 +235,16 @@ func k8sNameNamespace(coreName string, scope k8s_model.Scope) (string, string, e
 	}
 }
 
+func splitLabelsAndAnnotations(coreLabels map[string]string) (map[string]string, map[string]string) {
```

Comment:
please add a comment for this function, something like:
> Kuma resource labels are generally stored on Kubernetes as labels, except "kuma.io/display-name". We store it as an annotation because the resource name on k8s is limited by 253 and the label value is limited by 63.